src/server/origin/summary/index.njk

```diff
@@ -0,0 +1,67 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+
+{% extends 'layouts/questions.njk' %}
+
+{% block questions %}
+  <h1 class="govuk-heading-l">{{pageTitle}}</h1>
+
+  {{ govukSummaryList({
+    classes: "govuk-!-margin-bottom-9",
+    rows: [
+      {
+        key: {
+          text: "Are you moving the cattle on or off your farm or premises?"
```

Comment:
Similar to the above, this feels ripe for consolidation into something you pull from the session state. 

The reason I'm saying state instead of component or model is because later questions will be dynamic - e.g.: in R2 or R3 we will be asking about dilution ratios of the specific disinfectant you're using.  At that stage, the question text isn't static for a given page, it's generated based on prior state.

As with the others, I'm happy to merge for functionality & then refactor to address.